### ISSUE TITLE
Add support for DepthWiseConv2D layers

### ISSUE THREAD

**Issue by charlesmartin14** [CalculatedContent/WeightWatcher#169]
Add support for DepthWiseConv2D layers from Keras

https://www.tensorflow.org/api_docs/python/tf/keras/layers/DepthwiseConv2D

For use on models like EfficientNet

https://ai.googleblog.com/2019/05/efficientnet-improving-accuracy-and.html

https://huggingface.co/docs/timm/models/efficientnet

**Comment by charlesmartin14** [comment]
This should be working now.  But the layer is simply denoted Conv2D
Ill leave this open for a bit while we look into EfficientNet more
